CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_unified_rag v0.1.0-a7
+
+## Bug Fixes 
+- Fixed an issue in which [unioned](https://github.com/fivetran/dbt_unified_rag?tab=readme-ov-file#union-multiple-connections) source connections were producing null models.
+  - The solution required the addition of a base staging model layer. For each staging model, there is a `*_base` counterpart in  which we are running our `union_data` macro. This framework is necessary to the cooperation of our unioning and column-filling macros, which ensure the models do not fail if you are missing an expected column.
+  - For each connector type, this adds:
+    - **10 more models if Hubspot is enabled**
+    - **5 more models if Jira is enabled**
+    - **3 more models if Zendsk is enabled**
+
```

Comment:
@fivetran-avinash we also need to ensure the changes merged as a result of PR #23 are documented here as well.

@fivetran-jamie the changes in the above PR are a result of the consistency test failing. There was an issue in the previous pre-release version that has been addressed in the above PR that has not yet been deployed. This pre-release will include that change and we need to make sure we document it here.

Reply:
Thanks for the callout @fivetran-joemarkiewicz . 

@fivetran-jamie : If you want to add the CHANGELOG notes in, I think this should suffice:

"Updated `stg_rag_hubspot__owner` to correctly find columns from the `owner` source."

Reply:
added